Development Progress - week 06

rayapetkova opened the pull request:
Development Progress - week 06

karastoyanov committed:
test file

karastoyanov commented:
Workflow for the next week:

@rayapetkova will start working on Favorites menu implementation #57
@karastoyanov will continue working on the left layout buttons and fixing major issues #48 #50
@nikichhh and @DobromirMD will continue working on stylesheets #52

rayapetkova committed:
Update - query

karastoyanov committed:
adding to favorites button implemented

Co-Authored-By: Raya Petkova rayapetkova2007@abv.com

rayapetkova committed:
Updated favourite groupbox

- main_menu.py: when the button "favourites_button" is clicked it calls the open_favourites_func. This function returns open_favourites() func. It shows the new groupbox and hides the categories_groupbox.
- favourites.py: made an inner join query which gives us all the products that "pesho" has marked as "favourite". Made all the layouts and labels for each product.

rayapetkova committed:
update

karastoyanov committed:
Scroll favorites area implemented

karastoyanov committed:
check if favorite product already exists

karastoyanov committed:
left layout show/hide major fix
